examples/fscloud/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "ibmcloud_api_key" {
+  type        = string
+  description = "The IBM Cloud API token this account authenticates to"
+  sensitive   = true
+}
+
+variable "prefix" {
+  type        = string
+  description = "Prefix for sm instance"
+  default     = "secrets-manager-test"
+}
+
+variable "region" {
+  type        = string
+  description = "Region where resources will be created"
+  default     = "us-east"
+}
+
+variable "existing_resource_group" {
+  type        = bool
+  description = "Whether to use an existing resource group."
+  default     = false
+}
+
+variable "resource_group_name" {
+  type        = string
+  description = "A resource group name to use for this example, if `existing_resource_group` is false a new resource group will be created"
+  default     = "sm-fscloud"
+}
```

Comment:
can use be consistent with all other example and just have a variable called `resource_group`:

```
variable "resource_group" {
  type        = string
  description = "The name of an existing resource group to provision resources in to. If not set a new resource group will be created using the prefix variable"
  default     = null
}
```

Reply:
removed name